frontend/src/App.js

```diff
@@ -9,9 +10,11 @@ function App() {
           <div>Registration component</div>
         </Route>
         <Route path="/company-admin">
-          <div>Company</div>
+          <Menu />
+          <div id="page-content-wrapper">Company</div>
         </Route>
         <Route path="/super-admin">
+        <Menu />          
```

Comment:
Ez már majdnem jó, de jobb lenne, ha a menu-t egy helyen írnánk le valahogy, tehát mindig látszódjon, csak a login oldalon ne. Több switchet is lehet egymásba ágyazni, például a felső szintű csak aközött választana, hogy /login route vagy sem és ha nem, akkor azon belül lehetne egyszer a menü és mellette egy másik switch, ami már a többi oldalt routolja.